README.md

```diff
@@ -1,40 +1,91 @@
 # Machine API Operator
-An Operator for managing the cluster-api stack and the Openshift owned machineSets:
-- Aggregated API server
-- Controller manager
-- Machine controller (AWS/OpenStack/Libvirt actuator)
+The Machine API Operator manages the lifecycle of specific purpose CRDs, controllers and RBAC objects that extend the Kubernetes API.
+This allows to convey desired state of machines in a cluster in a declarative fashion.
 
-# Manual deployment (for Kubernetes cluster)
+## Architecture
+![Machine API Operator overview](machine-api-operator.png)
 
-When running the mao on the installer it assumes some existing resources that make it work as expected.
-To deploy the machine-api-operator in a vanilla kubernetes environment, one needs to precreate these assumptions:
+## CRDs
+- MachineSet
+- Machine
+- Cluster
+- MachineHealthCheck
 
-- Create the `openshift-machine-api-operator` namespace
-- Create the [CRD Status definition](tests/e2e/manifests/status-crd.yaml)
-- Create the [mao config](tests/e2e/manifests/mao-config.yaml)
-- Create the [apiserver certs](tests/e2e/manifests/clusterapi-apiserver-certs.yaml)
-- Create a given [ign config](tests/e2e/manifests/ign-config.yaml)
-- You can run it as a deployment with [this manifest](tests/e2e/manifests/operator-deployment.yaml)
+## Controllers
+### cluster-api-controllers
```

Comment:
You can drop the dashes, looks weird. `Cluster API controllers` looks more fancy.